<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>407</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>261</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="box1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="box2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="box3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="box4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="box5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="box6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="box7">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>180</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="box8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="box9">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>45</width>
       <height>45</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>289</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="labelPlayer">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Player: </string>
     </property>
    </widget>
    <widget class="QLabel" name="labelWinner">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonReset">
     <property name="geometry">
      <rect>
       <x>264</x>
       <y>20</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
